globalize.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'activerecord', '>= 4.2.0', '< 4.3'
   s.add_dependency 'activemodel', '>= 4.2.0', '< 4.3'
+  s.add_dependency 'request_store', '~> 1.1'
```

Comment:
Just to make it as easy as possible to migrate from Globalize 5.0 to Globalize 5.1. There are only some minor changes from 1.0 to 1.1 in RequestStore codebase. But feel free to leave it as is now.